tiledb/ml/data_apis/pytorch_sparse.py

```diff
@@ -31,6 +32,34 @@ def __init__(self, x_array: tiledb.Array, y_array: tiledb.Array, batch_size: int
         self.y = y_array
         self.batch_size = batch_size
 
+    def __check_row_dims(self, x_row_idx: np.array, y_row_idx: np.array):
+        """
+        Check the row dimensionality of x,y in case y is sparse or not
+
+        Parameters:
+            x_row_idx (list): Expects the row indexes x_coords of x Sparse Array of the
```

Comment:
Is it a list or a numpy array?

Reply:
It is a numpy! leftover I will correct it